### DISCUSSION TITLE
Why does the same code train much slower in pytorch than tensorflow

### DISCUSSION THREAD

**Discussion by ZZZ77QQQ** [lululxvi/deepxde#1574] [Q&A]
Hello, I would like to ask, I used the same code to run under the framework of tensorflow and pytorch respectively, and found that the training speed in pytorch is much slower than that in tensorflow, what is the reason?

**Comment by lululxvi** [comment]
Pytorch in general is slow.

**Reply by ZZZ77QQQ** [reply]
Thank you for your reply, I would like to ask a question, that is, when doing the inverse problem, if the parameter that you want to solve is not a fixed parameter but a variable parameter, is there a way to solve it

**Reply by yuluoxiansen** [reply]
Hello, for the sampling points in deepxde, can I take different numbers of points in area a and area b respectively? I want to take more points in a to better capture the characteristics of pde.

**Reply by lululxvi** [reply]
@ZZZ77QQQ Yes, there are demo examples.

**Reply by ZZZ77QQQ** [reply]
Thank you for your reply, I have another question to ask. In DeepONet, if the input of the trunk layer is a time series, is it necessary to normalize the time series?

**Reply by lululxvi** [reply]
Could be helpful.
